bin/setup-bench.mjs

```diff
@@ -0,0 +1,155 @@
+import 'zx/globals';
+import os from 'node:os';
+import { join } from 'node:path';
+
+/*
+
+  To run proper bench setup we need to do following things:
+
+  1.) Compile control packages
+  2.) Compile experiment packages
+  3.) Use SAME benchmark source
+      * we should be able to tweak bench
+        (add more cases, and still be able to compare with control)
+      * we should be able to re-run bench in CI from current branch with updated perf source
+
+*/
+
+const experimentBranchName =
+  process.env['EXPERIMENT_BRANCH_NAME'] || (await $`git rev-parse --abbrev-ref HEAD`).stdout.trim();
+const controlBranchName = process.env['CONTROL_BRANCH_NAME'] || 'main';
+const markers =
+  process.env['MARKERS'] ||
```

Comment:
is there any way to track all markers? or do we have to list them out? (I'm a tracerbench newb)

Reply:
@NullVoxPopuli we don't really need all markers (at least, in current step), it's confusing way to measure things, because we must know what we trying to measure.